resources/views/vendor/backpack/crud/columns/something.blade.php

```diff
@@ -0,0 +1,2 @@
+{{-- Used for 'custom_view' column example --}}
+<s>Something</s>
```

Comment:
Let's call this `custom_column_example`. the comment on the file helps. Nice touch 🙏 
 And maybe add an icon here too along with some text (maybe **half-bold**~~half-razored~~ ) to show that is a different flexible view.

Reply:
Done